lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -385,108 +379,99 @@ def _partition_pdf_batched_named(
         filename: str,
         hash_key: str,
         threshold: float = DEFAULT_LOCAL_THRESHOLD,
-        use_ocr=False,
-        ocr_images=False,
-        ocr_tables=False,
+        use_ocr: bool = False,
+        ocr_images: bool = False,
+        ocr_model: str = "easyocr",
+        per_element_ocr: bool = False,
         extract_table_structure=False,
         table_structure_extractor=None,
         extract_images=False,
         batch_size: int = 1,
         use_cache=False,
     ) -> List[List["Element"]]:
         self._init_model()
+
         if extract_table_structure and not table_structure_extractor:
             table_structure_extractor = DEFAULT_TABLE_STRUCTURE_EXTRACTOR(device=self.device)
-
+        if not use_ocr:
```

Comment:
With these two-branch simple conditionals, I prefer to see the positive case first.  It reads more easily without the not.  And hey, one word shorter!

Reply:
Done.